### DISCUSSION TITLE
Feedback for v2 - patches.lock filename

### DISCUSSION THREAD

**Discussion by Sweetchuck** [cweagans/composer-patches#485] [General]
Hello

I think the file name choice for `patches.lock` is not the best,
because by default none of the existing "text editors" or IDE provide syntax highlighting for this file.

I (and every developer) have to to re-configure all editors that I use (KATE, JEdit, Vim, PHPStorm, etc...) to recognize the `patches.lock` file as a regular JSON file.

**If the file content is a JSON then why not use the `.json` filename extension?**

I know that there is already `composer.lock` file name, which is already supported by most text editors.
In my humble opinion that was also a bad choice, but way too late for that to change it :-)

---

Suggestions:

* `patches.lock.json`
* `patches-lock.json`

**Comment by cweagans** [comment]
I would suggest that the contents of this file not be manipulated in a code editor, but I agree with this line of reasoning:

> If the file content is a JSON then why not use the .json filename extension?

I have opened https://github.com/cweagans/composer-patches/pull/492 to switch to `patches.lock.json`
